mkt/extensions/models.py

```diff
@@ -267,6 +267,7 @@ def mini_manifest(self):
         except ExtensionVersion.DoesNotExist:
             return {}
         mini_manifest = {
+            'id': self.uuid,
```

Comment:
We should mention in documentation endpoint that it's a UUID and not auto-incremented, just for posterity or whatever.